lib/rabbit_common/records.ex

```diff
@@ -17,7 +17,6 @@ defmodule RabbitCommon.Records do
   require Record
   import Record, only: [defrecord: 2, extract: 2]
 
-  defrecord :amqqueue, extract(:amqqueue, from_lib: "rabbit_common/include/rabbit.hrl")
```

Comment:
This is because `amqqueue` is [becoming private](https://github.com/rabbitmq/rabbitmq-common/pull/291) with the introduction of feature flags.

Reply:
Is it a question or do you answer another comment I didn't see?

Reply:
This is for those who are not sure why this record is no longer imported here.

Reply:
Ok, thank you.